README.md

```diff
@@ -286,6 +286,36 @@ Use this in one of your projects? Include one of these badges in your README.md
 [![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg?style=flat-square)](https://github.com/semantic-release/semantic-release)
 ```
 
+## Troubleshooting
+
+### ENOTINHISTORY Commit not in history
+
+```
+semantic-release ERR! commits The commit the last release of this package was derived from is not in the direct history of the "master" branch.
+semantic-release ERR! commits This means semantic-release can not extract the commits between now and then.
+semantic-release ERR! commits This is usually caused by force pushing, releasing from an unrelated branch, or using an already existing package name.
+semantic-release ERR! commits You can recover from this error by publishing manually or restoring the commit "123".
+semantic-release ERR! pre Failed to determine new version.
+semantic-release ERR! pre ENOTINHISTORY Commit not in history
+```
+
+To restore semantic-release, follow these steps:
+
+```
+git pull
+git reset --hard origin/master
+npm version x.y.z
+git checkout -B chore/release
+git commit -am 'chore: release'
+git push
+— merge (not squash-merge) on github (this is important before running git push)
+git checkout master
+git pull
+git push --tags
+npm publish
```

Comment:
I would suggest to leave out the pull-request flow and suggest to directly commit to master. If people care, they will know how to do it with pull requests.

I would suggest something like this

1. replace "0.0.0-development" in package.json with the next patch version of your `package.json` file. Find the current version with `npm info <package name> version`
2. run `npm publish`
3. revert the change to `package.json`
4. create a new tag with the new version, prefixed with `v`. So if the new version is `1.2.3`, run `git tag v1.2.3`
5. git push --tags
6. If you like, create a new release with the latest changes, as semantic release failed for the release. But it will work again from now on